Resources/Prototypes/Entities/Markers/Spawners/Random/Food_Drinks/food_breakfast.yml

```
@@ -0,0 +1,19 @@
+- type: entity
+  id: RandomFoodMeal
```

RandomFoodMeal already exists as an ID name. 
```suggestion
  id: RandomFoodBreakfast
```